### DISCUSSION TITLE
Price range filters on default search plugin

### DISCUSSION THREAD

**Discussion by twlite** [vendurehq/vendure#3077] [Q&A]
I saw `ElasticSearchPlugin` supporting price range filters i.e. `min` and `max` price but I do not see this in `DefaultSearchPlugin`. Is there a reason why `DefaultSearchPlugin` can't do price range filter?

**Comment by goranovs** [comment]
They most probbaly wont add that as it seems there is a **Paid** version of the Search Plugin with monthly subscription... WTF

https://vendure.io/hub/vendure-plus-advanced-search-plugin

**Reply by twlite** [reply]
I ended up modifying vendure and implemented it myself. It's not that difficult tbh, idk why it's not available by default

Edit: Also the plugin you mentioned seems to be backed by typesense, so we can't really compare it with default search plugin

**Reply by Vinod-Mendis** [reply]
Hey! I'm a beginner dev currently working on a project using Vendure, and I'm really interested in how you implemented this yourself. If you could share a small guide or walkthrough of your approach, that would be super helpful!

It’d be super helpful to understand your approach, especially since you mentioned it wasn’t too difficult to do.
